### PR TITLE
feat(profile): add user mutation

### PR DESCRIPTION
**Pull request by zerubeus**
# بسم الله الرحمن الرحيم

## What's new in this PR?

fixes

لله الحمد والمنة

### CODE REVIEW

File: src/services/apollo.ts [hunk 1]
```diff
@@ -1,46 +1,57 @@
...
+              graphQLErrors.forEach(({ message, locations, path }) =>
+                console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`)
+              )
+            if (networkError) console.log(`[Network error]: ${networkError}`)
+          }),
+          authLink.concat(httpLink),
+        ]),
+        cache: new InMemoryCache(),
+        typeDefs: gql`
+          enum user_genders_enum {
```
Comment (abumalick) [inline src/services/apollo.ts]: why do you need to define this? it is not in the schema?

**Review by abumalick — APPROVED** [review]
LGTM, بارك الله فيك

File: src/services/apollo.ts (hunk 1 above)
Comment (zerubeus) [reply src/services/apollo.ts]: It is in the schema but apollo don't pull the schema from the backend

Comment (zerubeus) [reply src/services/apollo.ts]: Indeed, it is not needed

**Pull request merged**